src/plugins/cft/api.js

```diff
@@ -0,0 +1,244 @@
+/**
+ * @author Ryadnov Andrey <me@ryadnov.ru>
+ */
+
+import {fetchJson} from "../../common/network";
+
+/**
+ * @type {boolean}
+ */
+const logFetches = true;
```

Comment:
is there some specific case where you'd actually want to disable network logging?
now all network logs go into console.debug context, which is filtered out by default in chrome console filters